cloud/deploys/bin/azure/db-connection

```diff
@@ -2,27 +2,31 @@
 source lib/out.sh
 source azure/bastion.sh
 
-# DOC: Connect to the database via the bastion
-# example: azure/db-connection -g sgdev -d civiform-artistic-halibut
-key_name="bastion"
-while getopts g:d:k flag
+# DOC: Connect to the database via the bastion. 
+# DOC: Required to pass in the resource group (-g), the database name (-d) and the keyvault name (-v). 
+# DOC: example: azure/db-connection -g sgdev -d civiform-artistic-halibut -v sgdev-civiform-kv
```

Comment:
suggestion (blocking): i think the help script only works if there's a single line that starts with `DOC: ` which is a little embarrassing 😅, please collapse these to a single line or fix the script

Reply:
i tried this and this lil hack does work!